vendor/DSCResources/README.md

```diff
@@ -0,0 +1,32 @@
+## DSC Resource Kit Wave 6
+
+This is a fixed version of the [DSC Resource Kit Wave 6](http://gallery.technet.microsoft.com/scriptcenter/DSC-Resource-Kit-All-c449312d)
+
+The fixes are for compatibility with the MOF parser. This is originally from https://github.com/msutter/dsc-resource-kit-wave-6
```

Comment:
Yup, temporarily. The mass update is https://tickets.puppetlabs.com/browse/FM-2564